packages/blinks-core/src/api/Action/action-supportability.ts

```diff
@@ -44,18 +45,21 @@ export type ActionSupportability =
 
 export type ActionSupportStrategy = (
   action: Action,
+  adapter?: ActionAdapter,
```

Comment:
I think we changing the behaviour when keep it optional here

Can you please doublecheck that e.g. monad blink shows incompatible fallback? 

solana-action:https://test-action-inputs.dialect.workers.dev/monad-poc/transfer

Reply:
will test, thank you!